packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -506,9 +506,9 @@ function TableVirtualizer({layout, collection, lastResizeInteractionModality, fo
           onScrollEnd={state.endScrolling}
           onScroll={onScroll}>
           {state.visibleViews[1]}
-          <div
+          {columnState.currentlyResizingColumn && <div
```

Comment:
If this is a key, you'll need to check if it's null and not only falsy

Reply:
haha, it got me!